notebook2/notebook/notebook.py

```diff
@@ -461,31 +501,39 @@ def auth0_callback():
     userinfo = auth0.get('userinfo').json()
 
     email = userinfo['email']
+    workshop_user = session.get('workshop_user', False)
 
-    if AUTHORIZED_USERS.get(email) is None:
+    if AUTHORIZED_USERS.get(email) is None and workshop_user is False:
```

Comment:
As mentioned below: This simply reverts pre-IBG2019 behavior: the user needs to have both an auth0 account and a workshop password, if they are not whitelisted (spoke to @cseed about this last week, but if I misunderstood, happy to revert).